.github/workflows/build.yaml

```diff
@@ -0,0 +1,77 @@
+name: build-app
+on: [push]
+jobs:
+  yarn-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - run: yarn install
+      - run: yarn build 
+
+  bump-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.output.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}   # checkout the correct branch name
+          fetch-depth: 0                # fetch the whole repo history
+      - id: setversion
+        uses: codacy/git-version@2.4.0
+        name: Git Version
+        with:
+          release-branch: main
+          dev-branch: dev
+      - id: output
+        run: echo "::set-output name=version::${{ steps.setversion.outputs.version }}"
+
+  tag-commit:
+    runs-on: ubuntu-latest
+    needs: bump-version
+    if: github.ref == 'refs/heads/chain4travel'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create Git tag
+        uses: actions/github-script@v4
+        with:
+          script: |
+            github.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: "refs/tags/${{ needs.bump-version.outputs.version }}",
+              sha: context.sha
+            })
+  docker-build:
+    runs-on: ubuntu-latest
+    needs: bump-version
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/login-action@v1
+        name: Login to DockerHub
+        with:
+          username: ${{ secrets.docker_username }}
+          password: ${{ secrets.docker_pass }}
+      - name: Set tag variables
+        id: setTagVariables
+        run: |
+          BRANCH_NAME=${{ github.ref_name }}
+          if [[ "${{ github.ref_name }}" == "main" ]]; then
+              echo "::set-output name=stable::c4tplatform/camino-docs:stable"
+          fi
+          if [[ "${{ github.ref_name }}" == "dev" ]]; then
+              echo "::set-output name=stable::c4tplatform/camino-docs:dev"
```

Comment:
I'm not familiar with the syntax here, but is "name=stable" right for the dev deploy?

Reply:
yes, as it just a variable name that is used in the build and push action